rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/SendMessageDirectives.scala

```diff
@@ -30,66 +29,30 @@ trait SendMessageDirectives {
 
         validateMessageAttributes(params.MessageAttributes.getOrElse(Map.empty))
 
-        doSendMessage(queueActor, message).map { case (message, digest, messageAttributeDigest) =>
-          complete(SendMessageResponse(messageAttributeDigest, digest, None, message.id.id, message.sequenceNumber))
+        doSendMessage(queueActor, message).map {
+          case MessageSendingResult(message, digest, messageAttributeDigest, messageSystemAttributeDigest) =>
+            complete(SendMessageResponse(messageAttributeDigest, digest, messageSystemAttributeDigest, message.id.id, message.sequenceNumber))
         }
       }
     }
   }
 
-  def getMessageAttributes(parameters: Map[String, String]): Map[String, MessageAttribute] = {
```

Comment:
Two functions were doing basically same job with parsing arguments -> one for message system attributes and other for attributes. I remove one and generalized other.